src/pages/PaymentPage/ConfirmedDetails.jsx

```diff
@@ -0,0 +1,79 @@
+import "./ConfirmedDetails.scss";
+import React , {useState , useEffect} from "react";
+import ApplyCoupon from "../../assets/icons/ApplyCoupon";
+import { useSelector } from "react-redux";
+
+function ConfirmedDetails() {
+  const [price , setPrice] = useState();
+
+  const { cartItems } = useSelector((state) => state.cart);
+
+  const totalAmount = () =>{
+    let total = 0;
+    cartItems.forEach((item) => {
```

Comment:
This code is redundant. Saw in some other component as well. Can you please check if it is possible to be at only one place?

Reply:
Fixed the issue and removed the redundant code.